### DISCUSSION TITLE
wire:target group

### DISCUSSION THREAD

**Discussion by amshehzad** [livewire/livewire#5765] [3. Feature ideas]
Is there a possibility to add a `wire:target` to a group of elements? something like `wire:target.group` or something e.g add `wire:target` to parent element and all elements within that have the same target
for example, have a look on this example:

```
<button
    wire:loading.attr="disabled" wire:target="addQuestion" wire:click="addQuestion"
    class="custom-btn text-m_regular mt-4"
>
    <span wire:loading.remove wire:target="addQuestion">Submit</span>
    <span wire:loading wire:target="addQuestion">Submitting...</span>
</button>
```

so here instead of adding `wire:target="addQuestion"` to 3 elements, we could add it on button and then all 3 elements have same target.

**Comment by lrljoe** [comment]
As discussed on Discord (for anyone else looking).

Your best bet would be a Blade Component for a button (let's call it button.blade.php), you can then use the attribute bag as you wish.

The below example discounts things like adding classes etc to your button component, purely for simplicity's sake
```
<button wire:loading.attr="disabled"  class="custom-btn text-m_regular mt-4" {{ $attributes }} >
    <span wire:loading.remove {{ $attributes->only('wire:target') }} >Submit</span>
    <span wire:loading {{ $attributes->only('wire:target') }}>Submitting...</span>
</button>
```
Then when calling
```
<x-button wire:target="addQuestion" wire:click="addQuestion" />
```
You'll note that the two "span" elements, also receive the wire:target parameter
